src/components/tables/TableHomeworkRow.js

```diff
@@ -0,0 +1,63 @@
+import { updateHomeworkThunk } from "../../store/slices/homeworksSlice";
+import { buildTask, findCurrentStudyYear } from "../../utils/services";
+import { useSelector } from "react-redux";
+import { useDispatch } from "react-redux";
+import moment from "moment/min/moment-with-locales.min";
+
+const TableHomeworkRow = ({ currentNumber, lessonItem }) => {
+  const { lessons } = useSelector((state) => state.lessons);
+  const homework = useSelector(
+    (state) => state.homeworks.homeworksList[lessonItem.homework]
+  );
+  const userId = useSelector((state) => state.user.id);
+  const currentStudyYear = findCurrentStudyYear(moment());
+  const dispatch = useDispatch();
+
+  const isAttenshion = () => {
+    if (lessonItem.lessonId && homework && !homework.isDone)
```

Comment:
we have cool stuff nowadays :)
```suggestion
    if (lessonItem.lessonId && !homework?.isDone)
```